src/main/java/uk/gov/hmcts/cmc/claimstore/rpa/config/EmailProperties.java

```diff
@@ -13,8 +13,18 @@
 
     @NotBlank
     private String sender;
+
+    @NotBlank
+    private String sealedClaimRecipient;
```

Comment:
Please change annotation at the class level to `@ConfigurationProperties(prefix = "rpa.notifications")` and the following variables:

```
RPA_NOTIFICATIONS_SENDER
RPA_NOTIFICATIONS_SEALED_CLAIM_RECIPIENT
RPA_NOTIFICATIONS_MORE_TIME_REQUESTED_RECIPIENT
RPA_NOTIFICATIONS_DEFENCE_RESPONSE_RECIPIENT
RPA_NOTIFICATIONS_COUNTY_COURT_JUDGEMENT_RECIPIENT
```

Sprint Boot 2.0 dropped support for `-` in property names.

Reply:
Changed.

Reply:
Partially changed.

Reply:
Already answered before.